src/hotspot/share/opto/vectornode.cpp

```diff
@@ -1127,6 +1143,41 @@ Node* StoreVectorMaskedNode::Ideal(PhaseGVN* phase, bool can_reshape) {
   }
   return StoreVectorNode::Ideal(phase, can_reshape);
 }
+Node* StoreVectorMaskedNode::Identity(PhaseGVN* phase) {
+  Node* value = StoreVectorNode::Identity(phase);
+  if ((value != this) &&  (value->is_StoreVectorMasked()) && (in(MemNode::ValueIn + 1)->eqv_uncast(value->in(MemNode::ValueIn + 1)))) {
```

Comment:
`MemNode::ValueIn + 1` is there no direct value for it? If not: maybe create one?

Reply:
Thanks for the comments @eme64. 
I've created one more constant for the "+ 2" case (`OffsetsMask`) and reused the one there for the "+ 1" case (`OopStore`).